symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/config/BdkDatafeedConfig.java

```diff
@@ -24,6 +25,7 @@
 /**
  * Injection of the {@link DatafeedService} instance into the Spring application context.
  */
+@ConditionalOnProperty(value = "bdk.datafeed.enabled", havingValue = "true", matchIfMissing = true)
 public class BdkDatafeedConfig {
```

Comment:
just found out we have too classes named BdkDatafeedConfig in BDK 2.0 maybe we should rename one?

Reply:
Classes in package `com.symphony.bdk.core.config.model` could be suffixed by `Model`, giving `com.symphony.bdk.core.config.model.BdkDatafeedConfigModel`